src/org/zaproxy/zap/extension/brk/impl/http/BreakAddEditDialog.java

```diff
@@ -146,6 +146,9 @@ public String validateFields() {
 			}
 		}
 		return null;
+		if(FIELD_STRING.contains("#")==true){
```

Comment:
- It should do the contains on the value of the field (e.g. `getStringValue(FIELD_STRING)`).
 - The comparison to `true` could be removed.
 - Note that this check needs to be done before the above return statement.